File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    var s=[]
    var e=[]
    var temp=[]
    for(let i=0; i< sizes.length; i++){
        temp.push(sizes[i].sort((a,b)=>b-a))
        s.push(temp[i][0])
        e.push(temp[i][1])
    }
    return Math.max(...s) *Math.max(...e)
}

File: 프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    max_long_side = 0
    max_short_side = 0

    for w, h in sizes:
        long_side = max(w, h)
        short_side = min(w, h)

        if long_side > max_long_side:
            max_long_side = long_side

        if short_side > max_short_side:
            max_short_side = short_side

    return max_long_side * max_short_side